tests/backup/backup_test.go

```diff
@@ -5256,48 +5512,47 @@ func AddRoleAndAccessToUsers(users []string, backupNames []string) (map[userRole
 		}
 		log.Infof(" Backup %s shared with user %s", backupNames[i], users[i])
 	}
-	return RoleAccessUserBackupContext, nil
+	return roleAccessUserBackupContext, nil
 }
-
-func ValidateSharedBackupWithUsers(key userRoleAccess, backupName string, restoreName string) {
+func ValidateSharedBackupWithUsers(user string, userCtx context.Context, access BackupAccess, backupName string, restoreName string) {
 	ctx, err := backup.GetAdminCtxFromSecret()
 	log.FailOnError(err, "Fetching px-central-admin ctx")
 	log.InfoD("Registering Source and Destination clusters from user context")
-	CreateSourceAndDestClusters(orgID, "", "", key.context)
-	log.InfoD("Validating if user [%s] with access [%v] and [%v] role can restore and delete backup or not", key.user, backupAccessKeyValue[key.accesses], key.roles)
+	CreateSourceAndDestClusters(orgID, "", "", userCtx)
```

Comment:
Shouldn't this func also return some errors if it failed?

Reply:
Yes Nikolay You are correct .
These are the changes we are addressing in future PRs

Reply:
done